src/router/index.ts

```diff
@@ -1,17 +1,101 @@
 import h from "hyperscript"
 import cx from "classnames"
+import { ROUTES } from "~router/routes"
+import { AppEvent, dispatch } from "~pages/app/events"
+import { rootElement } from "~index"
 
 export type LinkProps = {
   path: string
   title: string
   className?: string
+  children: Node | Node[]
 }
-export function Link({ path, title, className }: LinkProps) {
+export function Link({ path, title, children, className }: LinkProps) {
```

Comment:
What does this Link() method & where is it used?

Reply:
It's not currently used, but from my experience there is usually need for an `<a />` that triggers the router